plugin/trino-phoenix/src/main/java/io/trino/plugin/phoenix/PhoenixSplit.java

```diff
@@ -29,16 +29,16 @@
         extends JdbcSplit
 {
     private final List<HostAddress> addresses;
-    private final WrappedPhoenixInputSplit phoenixInputSplit;
+    private final SerializedPhoenixInputSplit serializedPhoenixInputSplit;
```

Comment:
nit: drop `serialized` from arg names a field name. Just leave `phoenixInputSplit`

Reply:
I named it with the `serialized` prefix to avoid name clash on getters (`getSerializedPhoenixInputSplit` and `getPhoenixInputSplit`)